org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -77,7 +77,7 @@ class Translator {
                 var hashFromPayload = this._generateHmac(verification.key, payloadAsString);
 
                 if (hashFromWink !== hashFromPayload) {
-                    throw new OpenT2TError(401, "Notification signature doesn't match.  Expecting " + hashFromWink + " and calculated " + hashFromPayload);
+                    throw new OpenT2TError(401, OpenT2TConstants.HMacSignatureVerificationFailed);
```

Comment:
What you can do here is to concatenate the constant with the "Expecting" part;
And the test in the cloud repo can then be updated with looking for (contains()) OpenT2TConstants.HmacSignatureVerificationFailed.